kroxylicious/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -85,7 +87,8 @@ public KafkaProxy(Configuration config) {
         this.logFrames = config.proxy().logFrames();
         this.useIoUring = config.proxy().useIoUring();
         this.adminHttpConfig = config.adminHttpConfig();
-
+        MicrometerConfiguration micrometerConfig = config.micrometerConfig();
+        this.micrometerConfig = micrometerConfig == null ? MicrometerConfiguration.empty() : micrometerConfig;
```

Comment:
I think the `null` check and using `MicrometerConfiguration.empty()` should be pushed down into `config.micrometerConfig`

Reply:
done